src/Compilers/Core/Portable/Syntax/SyntaxToken.cs

```diff
@@ -634,9 +634,12 @@ internal SyntaxToken GetPreviousToken(Func<SyntaxToken, bool> predicate, Func<Sy
         /// </summary>
         public Location GetLocation()
         {
-            return Node != null
-                ? this.SyntaxTree.GetLocation(this.Span)
-                : Location.None;
+            if (this.Node == null || this.SyntaxTree == null)
```

Comment:
Checked the existing initializers. Node cannot exist without a SyntaxTree. Will edit.

---
In reply to: [106474900](https://github.com/dotnet/roslyn/pull/17899#discussion_r106474900) [](ancestors = 106474900)